use smaller font sizes on Mac to get an experience closer to that on other platforms (and waste less space)

diff --git a/src/widgets/NumericTextCtrl.cpp b/src/widgets/NumericTextCtrl.cpp
index da0df2a21c3d376b3f62b909d65edb6746b9a3ab..628ebe2b405f659ffe87f8ef79813070736ddc04 100644
--- a/src/widgets/NumericTextCtrl.cpp
+++ b/src/widgets/NumericTextCtrl.cpp
@@ -1388,8 +1388,8 @@ NumericTextCtrl::NumericTextCtrl(wxWindow *parent, wxWindowID id,
 {
    mAllowInvalidValue = false;
 
-   mDigitBoxW = 11;
-   mDigitBoxH = 19;
+   mDigitBoxW = 8;
+   mDigitBoxH = 14;
 
    mBorderLeft = 1;
    mBorderTop = 1;
diff --git a/src/widgets/Ruler.cpp b/src/widgets/Ruler.cpp
index 402d5cb488a7cc9b34e5d233ce319a348226e212..11be58f2f73796967c996c7fc058eb8b027fe6bb 100644
--- a/src/widgets/Ruler.cpp
+++ b/src/widgets/Ruler.cpp
@@ -1002,14 +1002,14 @@ static constexpr int MinPixelHeight =
 #ifdef __WXMSW__
    12;
 #else
-   10;
+   7;
 #endif
 
 static constexpr int MaxPixelHeight =
 #ifdef __WXMSW__
    14;
 #elif __WXMAC__
-   10;
+   9;
 #else
    12;
 #endif
